contracts/interfaces/IDecentAutonomousAdmin.sol

```diff
@@ -0,0 +1,16 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.28;
+
+import {IHats} from "./hats/full/IHats.sol";
+import {ISablierV2Lockup} from "./sablier/full/ISablierV2Lockup.sol";
```

Comment:
Does this import need to be here? It's not being referenced in this interface file, so I would say this is probably not the right place for it.

Reply:
fixed